Node (#8)
belvo-finance/quickstart: README.md

BELVO_SECRET_PASSWORD=[YOUR_PASSWORD_HERE]
# You can choose between sandbox and production
BELVO_ENV=sandbox

> # You can choose between Python, Ruby and Node.js

Reviewer: The choice must match the docker image tag

```suggestion
# You can choose between python, ruby or node
```
